linux_os/guide/services/xwindows/disabling_xwindows/xwindows_remove_packages/oval/shared.xml

```diff
@@ -1,24 +1,25 @@
+
+{{% if product in ["sle12", "sle15"] %}}
+{{% set xwindows_packages = ['xorg-x11-server', 'xorg-x11-server-extra', 'xorg-x11-server-Xvfb', 'xwayland'] %}}
```

Comment:
Now that you made this variable at product level, this is not required

Reply:
Thanks :bow: should be handled in [aeb38eb](https://github.com/ComplianceAsCode/content/pull/12853/commits/aeb38eb0aa62316ccd418bafe7b1009e688f0cf0)